src/main/java/org/cryptomator/common/mount/MountPointPreparationException.java

```diff
@@ -1,12 +1,18 @@
 package org.cryptomator.common.mount;
 
+/**
+ * Used for wrapping exceptions which occur while preparing the mountpoint.<br>
+ * Instances of this exception are usually treated as generic error.
+ *
+ * @see IllegalMountPointException IllegalMountPointException for indicating configuration errors.
+ */
```

Comment:
DO we actually still need this exception? If this is for the generic error screen, we can then just throw the regarding IOExceptions, because the stacktrace will tell us, where it failed.

Reply:
It's not necessarily needed. I considered it worth keeping to create a more consistent internal API surface.
It's not explicitly caught anywhere, but is used to primarily wrap IOExceptions.

Reply:
Then away with it!

![ezgif com-add-text](https://github.com/cryptomator/cryptomator/assets/9036915/5467f4a2-f552-4f58-8a63-afa847be3c0e)

Reply:
Resolved by c3ac043c68d4add2c2ef687f34b541219cca81e2.